packages/docusaurus-theme-openapi/src/theme/ParamsItem/index.js

```diff
@@ -0,0 +1,66 @@
+/* ============================================================================
+ * Copyright (c) Cloud Annotations
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ * ========================================================================== */
+
+import React from "react";
+
+import { createDescription } from "../../../../docusaurus-plugin-openapi/src/markdown/createDescription";
```

Comment:
You should be able to import from `@paloaltonetworks/docusaurus-plugin-openapi` in place of `../../../../`.

Reply:
Ahh yes indeed. Thanks for the input!